util/health/testdata/pod-running-restart-onfailure.yaml

```diff
@@ -75,10 +75,6 @@ status:
     name: main
     ready: false
     restartCount: 4
-    state:
-      waiting:
-        message: Back-off 1m20s restarting failed container=main pod=my-pod_argocd(3cf9325e-f617-11e8-a057-fe5f49266390)
-        reason: CrashLoopBackOff
```

Comment:
We need to keep this state here. The expectation is a pod in this state is Progressing and not Degraded.